src/submission/forms.py

```diff
@@ -86,6 +86,7 @@ def __init__(self, *args, **kwargs):
         elements = kwargs.pop('additional_fields', None)
         submission_summary = kwargs.pop('submission_summary', None)
         journal = kwargs.pop('journal', None)
+        self.backend = kwargs.pop('backend', False)
```

Comment:
I think this can just be a local variable instead of an instance attribute.
Can we add a docstring explaining what it means? Or maybe use a name more specific to the form instead of specific to the caller e.g (`pop_disabled_fields`)

Reply:
Setting it as an instance attribute means I can also use it in the save method (which we do). I can change the variable name and add a docstring though!